api/include/mfxstructures.h

```diff
@@ -1929,9 +1929,20 @@ typedef struct {
 
 typedef struct {
     mfxExtBuffer Header;
-
     mfxU16 Enable;        /* tri-state option */
-    mfxU16 reserved[11];
+    union {
+        mfxU16   Y;
+        mfxU16   R;
+    };
+    union {
+        mfxU16   U;
+        mfxU16   G;
+    };
+    union {
```

Comment:
As a new uAPI this needs to be wrapped in VERSION_NEXT here and everywhere you use it.